product_state/data/product_state_data.xml

```diff
@@ -9,6 +9,7 @@
         <field name="code">sellable</field>
         <field name="name">Normal</field>
         <field name="sequence">20</field>
+        <field name="default">True</field>
```

Comment:
this file needs to be moved to the demo section in manifest

Reply:
Thanks for review, I will check it out !

Reply:
We had moved it to demo section but then need to revert back due to comments, [this comment](https://github.com/OCA/product-attribute/pull/931#issuecomment-966268076)

Reply:
thanks for the ansver